content/en/docs/reference/glossary/group-version-resource.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Group, Version, Resource
+id: gvr
+date: 2023-07-24
+short_description: >
+  The API group, API version and name of a Kubernetes API. 
+
+aka: ["GVR"]
+tags:
+- architecture
+---
+Means of representing unique Kubernetes API resource.
+
+<!--more-->
+
+Group, Version, Resources (GVRs) specify the API group, API version, and resource (name for the object kind as it appears in the URI) associated with accessing a particular id of object in Kubernetes.
```

Comment:
nit: The `id` is `gvr`, should the title use `Group Version Resource`?
nit: Group Version Resource (GVRs) specify the API group, version and resource ...

Reply:
Done.